src/smart-components/user/users.js

```diff
@@ -0,0 +1,46 @@
+import React, {  useState } from 'react';
+import {  Stack, StackItem } from '@patternfly/react-core';
+import { TopToolbar, TopToolbarTitle } from '../../presentational-components/shared/top-toolbar';
+import { Section } from '@redhat-cloud-services/frontend-components';
+import UsersList from '../group/add-group/users-list';
+
+const Users = () => {
+  const [ selectedRows, setSelectedRows ] = useState([]);
```

Comment:
If we are not going to use the state in here, no need to pretend it like this. Rather use `() => undefined` in `setSelectedUsers` prop.